LICENSE

```diff
@@ -1,80 +1,16 @@
-    consent of Neural Magic in each instance or use the Software on behalf of any third party. 
+nm-vllm is an enterprise inferencing system for deployments of open-source large language 
+models (LLMs) on GPUs. 
 
-    3. 	No Warranty. The Software is provided to you "AS IS" and without warranty. You are not 
-    entitled to any hard copy documentation, maintenance, support or updates for the Software 
-    unless you have entered into a separate commercial agreement under which support 
-    obligations are specified.
-
-    WE EXPRESSLY DISCLAIM ALL WARRANTIES RELATED TO THE SOFTWARE, EXPRESS OR IMPLIED, 
-    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
-    A PARTICULAR PURPOSE. WE DO NOT WARRANT THAT THE FUNCTIONS CONTAINED IN THE SOFTWARE WILL 
-    MEET YOUR REQUIREMENTS, OR THAT THE OPERATION OF THE SOFTWARE WILL BE UNINTERRUPTED OR 
-    ERROR-FREE, OR THAT DEFECTS IN THE SOFTWARE WILL BE CORRECTED. WE DO NOT WARRANT OR MAKE 
-    ANY REPRESENTATIONS REGARDING THE USE OR THE RESULTS OF THE USE OF THE SOFTWARE OR RELATED
-    DOCUMENTATION IN TERMS OF THEIR CORRECTNESS, ACCURACY, RELIABILITY OR OTHERWISE. SOME 
-    JURISDICTIONS DO NOT ALLOW THE EXCLUSION OF IMPLIED WARRANTIES, SO PORTIONS OF THE ABOVE 
-    EXCLUSION MAY NOT APPLY TO YOU.
-
-    4. 	Limitation of Liability. In no event shall we be liable to you for any damages with 
-    respect to your use of the Software. Further:
-
-    UNDER NO CIRCUMSTANCES, INCLUDING NEGLIGENCE, SHALL WE BE LIABLE FOR ANY INCIDENTAL, 
-    SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES ARISING OUT OF OR RELATING TO THIS LICENSE, 
-    INCLUDING, BUT NOT LIMITED TO, DAMAGES RESULTING FROM ANY LOSS OF DATA CAUSED BY THE 
-    SOFTWARE. SOME JURISDICTIONS DO NOT ALLOW THE LIMITATION OF INCIDENTAL OR CONSEQUENTIAL 
-    DAMAGES SO THIS LIMITATION MAY NOT APPLY TO YOU.
-
-    5. 	Controlling Law and Severability. This Agreement shall be governed by the laws of the
-    United States and those of the Commonwealth of Massachusetts. If for any reason a court of 
-    competent jurisdiction finds any provision, or portion thereof, to be unenforceable, the 
-    remainder of this Agreement shall continue in full force and effect. Any dispute regarding 
-    this Agreement shall be resolved in either the state or federal courts in Massachusetts. 
- 
-    6. 	Complete Agreement. This Agreement constitutes the entire agreement between us with 
-    respect to the version of the Software license hereunder. No amendment to or modification 
-    of this Agreement will be binding unless in writing. 
- 
-Your rights under this Agreement will terminate automatically without notice if you fail to 
-comply with any term(s) of this Agreement.
+There is one edition:
+Enterprise: fully licensed for production, commercial applications with Neural Magic's MSLSA:
+https://neuralmagic.com/legal/master-software-license-and-service-agreement/
```

Comment:
Nit: I think it is best practice to include the main points of the license in the embedded file itself rather than linking, since links are ephemeral